.github/workflows/vale.yml

```diff
@@ -0,0 +1,21 @@
+name: reviewdog
+on:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
I'm pretty sure you don't have any `main` branch but a `master` one 😄 
However I'm not sure these 3 lines are useful: see my global comment of the review